app/services/affiliate-slots-targeting.json

```diff
@@ -1,43 +1,74 @@
 [
   {
-    "campaign": "fandom-test",
-    "category": "fandom-test",
+    "campaign": "disneyplus",
+    "category": "disney",
     "wikiId": [
-      1081891
-    ],
-    "query": [
-      "coffee"
+      374,
```

Comment:
Are all those for the same set of wiki ids? Then the first one will ever be taken - we should limit either wikis OR add query so more diverisive set of units are available.